shared/constants/alerts.ts

```diff
@@ -2,6 +2,7 @@ export enum AlertTypes {
   unconnectedAccount = 'unconnectedAccount',
   web3ShimUsage = 'web3ShimUsage',
   invalidCustomNetwork = 'invalidCustomNetwork',
+  stxMigration = 'stxMigration',
```

Comment:
With new code for smart transactions we aim to not use the `stx` shortcut in naming and use the full name instead (e.g. `smartTransactions` or `smartTransaction`) to make code clearer. Can you please do a little renaming everywhere you used the word `stx`, including file names? Thank you